azure-quantum/azure/quantum/cirq/service.py

```diff
@@ -137,11 +137,16 @@ def create_job(
         :rtype: azure.quantum.cirq.Job
         """
         # Get target
-        target = self.get_target(name=target)
+        _target = self.get_target(name=target)
+        if not _target:
+            target_name = target or self._default_target
+            raise RuntimeError(f"Could not find target '{target_name}'. \
```

Comment:
Is it possible to print the provider name in the error message as well? Users may not know which provider to add based on the missing target, although I guess if they looked closely, all targets have the provider the dot?

Reply:
Maybe we can say "Could not find target '{target_name}' for provider '{parsed_provider_id}."

Is there also a possibility that the provider is provisioned, but the target is not supported in the region? This sometimes happens with QIO, where FPGA is not supported in all regions. In those cases, will the target also be missing?

Reply:
Currently, it's possible to get target only from the name, so we don't require a provider ID here. So yes, in that case the target will also be missing.